docs/getting-started/README.md

```diff
@@ -0,0 +1,137 @@
+# Getting started with Triggers
+
+To get started with Triggers, let's put it to work building and deploying a real image. In the following guide we will use `Triggers` to handle a real GitHub webhook request to kickoff a PipelineRun.
+
+## Install dependencies
+
+Before we can use the Triggers project, we need to get some dependencies out of the way.
+
+  - [A Kubernetes Cluster](https://kubernetes.io/docs/setup/)
+    - For our purposes any publicly available k8s cluster will do, as long as it can produce a Service Loadbalancer.
```

Comment:
The service is no longer a LoadBalancer, but ClusterIP. When speaking with @dibyom, it should also be documented that ingress does not work out of the box on GKE. See: https://github.com/kubernetes/kubernetes/issues/26508#issuecomment-222376962

Reply:
We need a better solution long term (e.g. maybe let users specify the service type) but for now, we could probably use the nginx ingress: https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke

(I can add this bit in later once this PR is merged)

Reply:
Added a bit for this, ty. Im not super familiar with the ingress/GKY situation so any additional suggestions welcome! 😄